bash_utils/config.sh

```diff
@@ -139,8 +139,8 @@ function import_recipe {
     # Loading into Database
     run_sql_file $target_dir/${name}.sql
     run_sql_command \
-        "ALTER TABLE ${name} ADD COLUMN v text; \
-        UPDATE ${name} SET v = '${version}'; \
+        "ALTER TABLE ${name} ADD COLUMN data_library_version text; \
+        UPDATE ${name} SET data_library_version = '${version}'; \
```

Comment:
This was to avoid collision. These aren't used in export, just for dev QOL while troubleshooting so think it's fine